src/otx/engine/engine.py

```diff
@@ -155,6 +171,9 @@ def train(
         callbacks: list[Callback] | Callback | None = None,
         logger: Logger | Iterable[Logger] | bool | None = None,
         resume: bool = False,
+        run_hpo: bool = False,
+        hpo_time_ratio: int = 4,
+        hpo_cfg_file: str | Path | None = None,
```

Comment:
I think this should go in the form of `hpo_config`. I propose a way to make it easy for users to know the configuration values related to hpo. For example, I think it could be something like `src/otx/core/config/data.py::TilerConfig`.

Reply:
I'll try to change as you said.

Reply:
I changed it as you said at https://github.com/openvinotoolkit/training_extensions/pull/2912/commits/8114a6ce66feda223fc892e494b6e9c5fb984f2c